src/metabase/api/database.clj

```diff
@@ -1054,8 +1048,8 @@
   "Returns a list of all syncable schemas found for the database `id`."
   [id]
   {id ms/PositiveInt}
-  (api/check-superuser)
```

Comment:
Curious why this check was changed?

Reply:
Settings Managers aren't superusers but still need access to this endpoint. The new check below (`(mi/can-write? db)`) should include superusers.

Um, in light of that other PR you reviewed, should it actually be `(can-read? db)`?

Reply:
Re-reading the Slack thread, I think it makes sense to keep `can-write?` here since it's an admin task (and agree that `can-read?` should be used for normal users to be able to upload CSVs)